MathLib/LinAlg/PETSc/PETScVector.cpp

```diff
@@ -77,6 +77,17 @@ PETScVector::PETScVector(const PETScVector &existing_vec, const bool deep_copy)
     }
 }
 
+PETScVector::PETScVector(PETScVector &&other)
+    : _v{std::move(other._v)}
```

Comment:
if you ask me so, my answer is "yes i do". Loosing consistency is not very minor thing for me.